stdlib/ballerina-http/src/main/ballerina/ballerina/net/http/annotation.bal

```diff
@@ -54,7 +55,7 @@ public annotation configuration attach service<> {
     string[] allowHeaders;
     int maxAge;
     string[] exposeHeaders;
-    boolean keepAlive;
+    string keepAlive;
```

Comment:
Why can't we use enum instead of String here?

Reply:
Please check https://github.com/ballerina-lang/ballerina/blob/master/tests/ballerina-test/src/test/resources/test-src/enums/enums.bal

Reply:
then should the enum be defined somewhere so the user can use it?

Reply:
Yes. It should come as a part of http package which will look like ```http:KeepAlive.AUTO```

Reply:
Tracked by this issue for both chunking and keep-alive config https://github.com/ballerina-lang/ballerina/issues/4961